book/website/community-handbook/local-build.md

```diff
@@ -1,154 +1,177 @@
-   ```
-   ````
+```console
+$ python3 -m venv ./venv
+```
 
-6. Navigate into the cloned repository folder using the command `cd the-turing-way`, where the `cd` command means `change directory`.
+Next, active the virtual environment,
 
-7. Then change into the sub-directory the website is built from using `cd book/website`
+```console
+$ source ./venv/bin/activate
+```
 
-8. The Turing Way book is built using multiple python libraries. We can install these dependencies _into your conda environment_ using the following command
+Your prompt may now start with `(venv)`, for example `(venv) user@host$`.
+Using the virtual environment means we can install _The Turing Way's_ dependencies without interfering with other packages or libraries you might be using.
+That will be explained in more depth in [a later section](#why-we-recommend-using-a-virtual-environment).
 
-   ```console
-   pip install -r requirements.txt
-   ```
+### Install the Dependencies
 
-   where the `requirements.txt` file contains a list of python libraries
+The next steps use {term}`Makefile`.
+The Makefile contains instructions to build a set of "targets".
+That way we can easily run the same commands repeatedly, and in different environments, without needing to remember all the parameters.
+It is easiest to change into the directory containing the Makefile,
 
-9. And now build the book:
+```
+$ cd book
+```
 
-   ```console
-   jupyter-book build .
-   ```
+Install the build dependencies into your virtual environment,
 
-10. The output of the build process will provide output such as below that demonstrate how you can view the book locally:
+```console
+$ make deps
+```
 
-    ```text
-    ===============================================================================
+### Build the Book
 
-    Finished generating HTML for book.
-    Your book's HTML pages are here:
-        _build/html/
-    You can look at your book by opening this file in a browser:
-        _build/html/index.html
-    Or paste this line directly into your browser bar:
-        file:///[...]/the-turing-way/the-turing-way/book/website/_build/html/index.html
+You are now ready to build the book.
+You can build the book with,
 
-    ===============================================================================
-    ```
+```console
+$ make book
+```
 
-### Build the book while working on a Pull Request
+The output of the build process will provide output such as below that demonstrate how you can view the book locally,
 
-If you would like to preview a version of the book from a certain branch (perhaps to render the book while working on a PR) then simply switch to the required branch and rebuild the book as in step 9:
+```text
+===============================================================================
 
-   ```bash
-   git checkout mybranch
-   jupyter-book build .
-   ```
+Finished generating HTML for book.
+Your book's HTML pages are here:
+    _build/html/
```

Comment:
Should _build be in the `.gitignore` to help people avoid committing local build outputs to their repo?

Reply:
It is, just not in the root of the repository :rocket:

https://github.com/the-turing-way/the-turing-way/blob/main/book/.gitignore